example_scenes/manim.cfg

```diff
@@ -0,0 +1,12 @@
+[log.color]
+logging.keyword = magenta
+logging.level.notset = dim
+logging.level.debug = yellow
+logging.level.info = dim purple
+logging.level.warning = dim red
+logging.level.error = red
+logging.level.critical = red
+log.level = 
+log.time = dim yellow
+log.message = green
+log.path = dim blue
```

Comment:
Similarly, the convention in the rest of the file is to use underscores between words, and not dots `.`. I assume these dots are necessary for rich purposes? If so, can we please consider using e.g. `log_message` here instead of `log.message` and then replacing those underscores when/if needed? Just a suggestion!